src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -76,7 +76,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      //TODO: add Password
```

Comment:
sorry, didn't mean for the TODO comments to be removed as well - these ones can stay.

Reply:
Done